### PR TITLE
docs: add tailwind-css instructions

### PR DESCRIPTION
**Pull request by ChrisEdgington**

### CODE REVIEW

File: guides/framework-integration/tailwind.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+{% hint style="info" %}
+The following guide has been tested with Tailwind 3, and Webpack 5.
+{% endhint %}
+
+### Create the app and install Tailwind CSS
+
+Create the app with the [Webpack template](../../templates/webpack-template.md).
+
+Install Tailwind CSS using the [`postcss instructions`](https://tailwindcss.com/docs/installation/using-postcss).
```
Comment (erickzhao) [inline guides/framework-integration/tailwind.md:17]: IMO we might want to port the instructions over here

**Review by erickzhao — COMMENTED** [review]
The structure of the guide overall looks good to me. Just one comment! Will do another pass for copy editing.

File: guides/framework-integration/tailwind.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
+---
+description: How to create an Electron app with Tailwind and Electron Forge
+---
+
+# Tailwind CSS
+
+Adding Tailwind CSS support to the Webpack template involves adding postcss support to the webpack configuration.
```
Comment (malept) [inline guides/framework-integration/tailwind.md:7]: suggestion: link to other projects and capitalize as appropriate.
```suggestion
Adding [Tailwind CSS](https://tailwindcss.com/) support to an Electron Forge project using the [Webpack plugin](../../config/plugins/webpack.md) involves adding [PostCSS](https://postcss.org/) support to the Webpack configuration.
```

File: guides/framework-integration/tailwind.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+
+### Create the app and install Tailwind CSS
+
+Create the app with the [Webpack template](../../templates/webpack-template.md).
+
+Install Tailwind CSS using the [`postcss instructions`](https://tailwindcss.com/docs/installation/using-postcss).
+
+### Configure Webpack to use postcss
+
+Install the postcss-loader component:
```
Comment (malept) [inline guides/framework-integration/tailwind.md:21]: Nit(formatting):
```suggestion
Install the `postcss-loader` component:
```

File: guides/framework-integration/tailwind.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+{% endtab %}
+
+{% tab title="NPM" %}
+```text
+npm install --save-dev postcss-loader
+```
+{% endtab %}
+{% endtabs %}
+
+Modify the renderer webpack configuration to include the postcss-loader for its css rules in `webpack.renderer.config.js`:
```
Comment (malept) [inline guides/framework-integration/tailwind.md:37]: Nit(formatting):
```suggestion
Modify the renderer Webpack configuration to include the `postcss-loader` for its CSS rules in `webpack.renderer.config.js`:
```

File: guides/framework-integration/tailwind.md [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+
+{% code title="webpack.renderer.config.js" %}
+```javascript
+rules.push({
+  test: /\.css$/,
+  // This is the line that needs modified - by adding the postcss-loader to the list of css rules
```
Comment (malept) [inline guides/framework-integration/tailwind.md:43]: Nit(grammar):
```suggestion
  // Modify the following line by adding the postcss-loader module to the list of loaders for CSS files
```

File: SUMMARY.md [hunk 6]
```diff
@@ -52,6 +52,7 @@
   * [React](guides/framework-integration/react.md)
   * [React with TypeScript](guides/framework-integration/react-with-typescript.md)
   * [Vue 2](guides/framework-integration/vue-2.md)
+  * [Tailwind CSS](guides/framework-integration/tailwind.md)
```
Comment (malept) [inline SUMMARY.md:61]: suggestion: Please order the guides alphabetically.

File: guides/framework-integration/tailwind.md [hunk 7]
```diff
@@ -0,0 +1,67 @@
...
+    <p>Welcome to your Electron application.</p>
+  </body>
+```
+{% endcode %}
+
+For more about Tailwind CSS, see [their documentation](https://tailwindcss.com/docs).
+
+### Using with React and Typescript
+
+These instructions also work for react, typescript, and react + typescript projects. Follow these instructions after confirming that react and typescript are configured and rendering properly.
```
Comment (malept) [inline guides/framework-integration/tailwind.md:66]: suggestion: capitalize project names accordingly, fix up grammar, and link to the appropriate guides.
```suggestion
These instructions also work for the [TypeScript + Webpack template](../../templates/typescript-%2B-webpack-template.md)  and the [React](react.md) and [React + TypeScript](react-with-typescript.md) guides. Follow these instructions after confirming that TypeScript and/or React are configured and rendering properly. 
```

File: guides/framework-integration/tailwind.md [hunk 8]
```diff
@@ -0,0 +1,67 @@
...
+  <body>
+    <h1 class="text-3xl font-bold underline text-green-500">💖 Hello World!</h1>
+    <p>Welcome to your Electron application.</p>
+  </body>
+```
+{% endcode %}
+
+For more about Tailwind CSS, see [their documentation](https://tailwindcss.com/docs).
+
+### Using with React and Typescript
```
Comment (malept) [inline guides/framework-integration/tailwind.md:64]: Nit(capitalization):
```suggestion
### Using with React and TypeScript
```

File: guides/framework-integration/tailwind.md (hunk 1 above)
Comment (erickzhao) [inline guides/framework-integration/tailwind.md:17]:
````suggestion
Starting with the Electron Forge [Webpack Template](../../templates/webpack-template.md), install Tailwind CSS using the [PostCSS instructions](https://tailwindcss.com/docs/installation/using-postcss).

{% tabs %}
{% tab title="Yarn" %}
```bash
yarn add --dev tailwindcss postcss autoprefixer
npx tailwindcss init -p
```
{% endtab %}

{% tab title="npm" %}
```bash
npm install --save-dev tailwindcss postcss autoprefixer
npx tailwindcss init -p
```
{% endtab %}
{% endtabs %}

Running the `npx tailwindcss init -p` script will create new `tailwind.config.js` and `postcss.config.js` configs for you. After this code is initialized, make sure you include all source files in your Tailwind `content` config and the necessary `@tailwind` directives to your stylesheet.

{% tabs %}
{% tab title="tailwind.config.js" %}
```js
/** @type {import('tailwindcss').Config} */
module.exports = {
  content: ["./src/**/*.{html,js}"],
  theme: {
    extend: {},
  },
  plugins: [],
}

```
{% endtab %}

{% tab title="src/index.css" %}
```css
@tailwind base;
@tailwind components;
@tailwind utilities;
```
{% endtab %}
{% endtabs %}


````

**Comment by dsanders11** [comment]
Closing this as it's gone stale.

**Pull request closed without merging**